runtime/vm/resolvesupport.cpp

```diff
@@ -972,7 +972,26 @@ resolveInstanceFieldRefInto(J9VMThread *vmStruct, J9Method *method, J9ConstantPo
 		
 			if (ramCPEntry != NULL) {
 				UDATA valueOffset = fieldOffset;
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+				if ('Q' == *J9UTF8_DATA(signature)) {
+					J9FlattenedClassCache *flattenedClassCache = classFromCP->flattenedClassCache;
+					J9Class *flattenableClass = NULL;
+					UDATA index = 0;
```

Comment:
Last nitpick - declare and assign index.

Reply:
done